crates/lowering/tests/snapshots/lowering__module_const.snap

```diff
@@ -3,16 +3,47 @@ source: crates/lowering/tests/lowering.rs
 expression: lowered_code
 
 ---
-contract Foo:
-    pub fn bar() -> u256:
-        if true:
-            return 3 * 10
-        else true:
-            return 10
+struct $tuple_u256_u256_:
+    item0: u256
+    item1: u256
 
+struct Bar:
+    val: u256
 
-    pub fn baz() -> u256:
-        return 3 * 3
+contract Foo:
+    table: Map<u256, u256>
 
     pub fn return_unit() -> ():
         return ()
+
+    pub fn usage(self) -> u256:
+        let my_calc: u256 = 3 * 10
+        let my_other_calc: u256 = 3 * 3
+        let my_array: u256[2] = list_expr_array_u256_2(3, 10)
+        let my_tuple: $tuple_u256_u256_ = $tuple_u256_u256_(item0=3, item1=10)
+        let my_bar: Bar = Bar(val=3)
+        while 3 > 4:
+            pass
+
+        for x in list_expr_array_u256_2(3, 10):
+            pass
+
+        self.table[10] = 3
+        if THREE == TEN:
+            pass
+        else:
```

Comment:
It looks like from our AST there's no way to reconstruct the actual `elif` when we print the AST. Doesn't seem like a problem today but would become a problem when we want to have something like `fe fmt`.